pingora/examples/client.rs

```diff
@@ -12,29 +12,58 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+use anyhow::Error;
 use pingora::connectors::http::Connector;
 use pingora::upstreams::peer::HttpPeer;
 use pingora_http::RequestHeader;
```

Comment:
Some of these like `HttpPeer` and `Connector` you can replace with `pingora::prelude::*` (which can also be helpful when bringing in the pingora error types). (These examples predated the existence of the prelude.)

Reply:
:+1: 5833556e5f8d9ccb346ef586cb1b5f51add64357

(However, `Connector` is not available in the prelude so I imported that directly. Not sure if you want it added to the prelude at this time.)